test/controllers/admin/admin_controller_test.rb

```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
+
+class Admin::UsersControllerTest < ActionController::TestCase
```

Comment:
file named as `admin_controller_test.rb`
but test class is `UsersControllerTest`

Reply:
you should have only `test/controllers/admin/users_controller_test.rb`
place this code there